src/lib/plot/series/area-series.js

```diff
@@ -65,7 +65,8 @@ class AreaSeries extends AbstractSeries {
       this._getAttributeValue('color');
     const fill = this._getAttributeValue('fill') ||
       this._getAttributeValue('color');
-    const opacity = this._getAttributeValue('opacity') || DEFAULT_OPACITY;
+    const givenOpacity = this._getAttributeValue('opacity');
+    const opacity = !isNaN(givenOpacity) ? givenOpacity : DEFAULT_OPACITY;
```

Comment:
`isNaN` has some quirks with coercion and its behavior is quite confusing (see how it handles true/false/undefined). How about using `Number.isFinite`? This also avoids the double negation here.

Refs:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN